xen/arch/x86/slaunch.c

```diff
@@ -72,7 +99,7 @@ void __init map_slaunch_mem_regions(void)
         map_l2(get_slb_start(), SKINIT_SLB_SIZE);
     }
 
-    find_evt_log(__va(slaunch_slrt), &evt_log_addr, &evt_log_size);
+    find_evt_log(slr_get_table(), &evt_log_addr, &evt_log_size);
```

Comment:
Truncated commit title?

Reply:
No, just ambiguous.  Reworded in 775ba698e7b926a4e72e1bf3e93772892b0333fa.